### DISCUSSION TITLE
Need import block list importer exporter btw accounts (game: VRChat)

### DISCUSSION THREAD

**Discussion by plautus2** [community/community#143158] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

can someone make a client or  something where u can do this?

**Comment by LiteBrite82** [comment]
Hi @plautus2 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by plautus2** [comment]
where do I post this then:?"

**Comment by plautus2** [comment]
any on this  news?
